UnityProject/Packages/packages-lock.json

```diff
@@ -236,11 +236,11 @@
       "url": "https://packages.unity.com"
     },
     "org.unitystation.spacemirror": {
-      "version": "https://github.com/unitystation/Mirror.git?path=/Assets/Mirror#dev",
+      "version": "https://github.com/unitystation/Mirror.git?path=/Assets/Mirror#master",
       "depth": 0,
       "source": "git",
       "dependencies": {},
-      "hash": "2afb70b72c7d784faee0ba893b93147ac91f3dad"
+      "hash": ""
```

Comment:
I think I know why this is needed. The actual way to change the branch in ``manifest.json`` would be using the graphical interface of the package manager and not manually changing the json file.

Reply:
I'm leaving it blank so that It will pull the latest version